### PR TITLE
Remove duplicative instruction

### PR DESCRIPTION
**Pull request by jbritt1**
## Description of change

Removes duplication of https://github.com/department-of-veterans-affairs/vets-api/blob/master/Dockerfile#L22-L28 in https://github.com/department-of-veterans-affairs/vets-api/blob/master/Dockerfile#L89. 

Relates to: 
- https://dsva.slack.com/archives/CTYQL39FE/p1582303374138200
- https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/5627
- department-of-veterans-affairs/va.gov-team#5627
- https://github.com/department-of-veterans-affairs/devops/pull/6427

## Testing done

N/A (just a removal of duplicated action)

## Testing planned

Built and deployed to dev, which led to an issue with an Okta SSL connection. Went back to trying to spin the container locally, found the expression being used was not liking that multiple files were "found", so the rest of the command was not doing what we thought it was. Removing the conditional on the command worked locally. 

## Acceptance Criteria (Definition of Done)
- [x] Removal of duplicated task

#### Unique to this PR

- [x] _Replace this line with individual tasks unique to your PR_ - n/a

#### Applies to all PRs

- [x] Appropriate logging -n/a
- [x] Swagger docs have been updated, if applicable - n/a
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable) - n/a

### CODE REVIEW

**Comment by omgitsbillryan** [comment]
I think you've indirectly answered my above 2 questions ☝️ ... but I was hoping to be spoon-fed rather than have to go digging 🙂

**Comment by jbritt1** [comment]
So we good to go then? lol

**Comment by omgitsbillryan** [comment]
Lastly, we probably need to use some caution while deploying this to `prod`. Certs are always a pain and easy to get wrong.

At the least, merge this to master directly after a daily deploy to allow for time to test in `dev` & `staging`. At the most cautious, maybe a `prod` deploy during low-usage hours.

**Comment by wyattwalter** [comment]
@omgitsbillryan This was the initial implementation which worked until we revamped review instances. I introduced this new `config/ca-trust` dir in https://github.com/department-of-veterans-affairs/vets-api/pull/3856 because this particular implementation didn't work for review instances. We agreed in that PR to consolidate the two into a single one, so this is just cleaning up that TODO item.

The merge on this does need to wait until https://github.com/department-of-veterans-affairs/devops/pull/6427 has been merged and tested.

**Comment by jbritt1** [comment]
image

Seems the dir structure was created as intended and cert files are there. Now just gotta check the backend connectivity.

**Comment by jbritt1** [comment]
```
[root@ip-10-247-96-203 ~]# docker exec -it vets-api bundle exec rails c
bin/rails: No such file or directory - git
{"host":"32092dffcf3e","application":"vets-api-server","timestamp":"2020-02-25T21:13:34.706385Z","level":"info","level_index":2,"pid":214,"thread":"47336504400220","name":"Rails","message":"Raven 2.9.0 ready to catch errors"}
SHRINE DEPRECATION WARNING: The logging plugin has been deprecated in favor of instrumentation plugin. The logging plugin will be removed in Shrine 3.
SHRINE DEPRECATION WARNING: Using aws-sdk-s3 < 1.14 is deprecated and support for it will be removed in Shrine 3. Update to aws-sdk-s3 version 1.14 or higher
Loading production environment (Rails 5.2.4.1)
irb(main):001:0> Search::Service.new('homeless').results
=> #<Search::ResultsResponse:0x0000561ad20941b8 @body={"query"=>"benefits", "web"=>{"total"=>3, "next_offset"=>20, "spelling_correction"=>nil, "results"=>[{"title"=>"Apply for education εÇÇbenefitsεÇü", "url"=>"https://www.vets.gov/education/apply-for-education-benefits/application/1995/introduction", "snippet"=>"Use your VA education εÇÇbenefitsεÇü to pay for college or training programs. Find...documents youll need to apply for εÇÇbenefitsεÇü, and start your online application...Manage your health & εÇÇbenefitsεÇü Find VA locations Home Education Apply for Education...Education εÇÇBenefitsεÇü Manage your education εÇÇbenefitsεÇü Equal to VA Form 22-1995", "publication_date"=>nil}, {"title"=>"Education εÇÇBenefitsεÇü", "url"=>"https://www.vets.gov/education/", "snippet"=>"Find out which VA education εÇÇbenefitsεÇü you may qualify for through the GI Bill...Education εÇÇBenefitsεÇü We offer Veterans, Servicemembers, and their families education...education εÇÇbenefitsεÇü like help paying tuition, help finding the right school", "publication_date"=>nil}]}, "text_best_bets"=>[{"title"=>"Apply for education εÇÇbenefitsεÇü", "url"=>"https://www.vets.gov/education/apply-for-education-benefits/application/1995/introduction", "description"=>"Use your VA education εÇÇbenefitsεÇü to pay for college or training programs. Find...documents youll need to apply for εÇÇbenefitsεÇü, and start your online application...Manage your health & εÇÇbenefitsεÇü Find VA locations Home Education Apply for Education...Education εÇÇBenefitsεÇü Manage your education εÇÇbenefitsεÇü Equal to VA Form 22-1995", "id"=>1}], "graphic_best_bets"=>nil, "health_topics"=>nil, "job_openings"=>nil, "recent_tweets"=>[{"text"=>"What is VA compensation? Many people donΓÇÖt fully understand the intent, purpose, or process behind this εÇÇbenefitεÇü. HeΓÇª https://t.co/3OWDjaEqh6", "url"=>"https://twitter.com/VAVetBenefits/status/1040228111646949377", "name"=>"Veterans Benefits", "snippet"=>nil, "screen_name"=>"VAVetBenefits", "profile_image_url"=>"https://pbs.twimg.com/profile_images/344513261572743396/a9fcce7feb947b2ec498491c6c6d6985_normal.png"}], "recent_news"=>nil, "recent_video_news"=>nil, "federal_register_documents"=>nil, "related_search_terms"=>nil}, @status=200, @pagination={"current_page"=>1, "per_page"=>10, "total_pages"=>1, "total_entries"=>3}, @metadata={}, @errors_hash={}, @original_attributes={:status=>nil, :body=>{"query"=>"benefits", "web"=>{"total"=>3, "next_offset"=>20, "spelling_correction"=>nil, "results"=>[{"title"=>"Apply for education εÇÇbenefitsεÇü", "url"=>"https://www.vets.gov/education/apply-for-education-benefits/application/1995/introduction", "snippet"=>"Use your VA education εÇÇbenefitsεÇü to pay for college or training programs. Find...documents youll need to apply for εÇÇbenefitsεÇü, and start your online application...Manage your health & εÇÇbenefitsεÇü Find 
VA locations Home Education Apply for Education...Education εÇÇBenefitsεÇü Manage your education εÇÇbenefitsεÇü Equal to VA Form 22-1995", "publication_date"=>nil}, {"title"=>"Education εÇÇBenefitsεÇü", "url"=>"https://www.vets.gov/education/", "snippet"=>"Find out which VA education εÇÇbenefitsεÇü you may qualify for through the GI Bill...Education εÇÇBenefitsεÇü We offer Veterans, Servicemembers, and their families education...education εÇÇbenefitsεÇü like help paying tuition, help finding the right school", "publication_date"=>nil}]}, "text_best_bets"=>[{"title"=>"Apply for education εÇÇbenefitsεÇü", "url"=>"https://www.vets.gov/education/apply-for-education-benefits/application/1995/introduction", "description"=>"Use your VA education εÇÇbenefitsεÇü to pay for college or training programs. Find...documents youll 
need to apply for εÇÇbenefitsεÇü, and start your online application...Manage your health & εÇÇbenefitsεÇü Find VA locations Home Education Apply for Education...Education εÇÇBenefitsεÇü Manage your education εÇÇbenefitsεÇü Equal to VA Form 22-1995", "id"=>1}], "graphic_best_bets"=>nil, "health_topics"=>nil, "job_openings"=>nil, "recent_tweets"=>[{"text"=>"What is VA compensation? Many people donΓÇÖt fully understand the intent, purpose, or process behind this εÇÇbenefitεÇü. HeΓÇª https://t.co/3OWDjaEqh6", "url"=>"https://twitter.com/VAVetBenefits/status/1040228111646949377", "name"=>"Veterans Benefits", "snippet"=>nil, "screen_name"=>"VAVetBenefits", "profile_image_url"=>"https://pbs.twimg.com/profile_images/344513261572743396/a9fcce7feb947b2ec498491c6c6d6985_normal.png"}], "recent_news"=>nil, "recent_video_news"=>nil, "federal_register_documents"=>nil, "related_search_terms"=>nil}, :pagination=>nil}>
irb(main):002:0>
```

**Comment by jbritt1** [comment]
The above output was from a deployed instance using the changes in [this pr](https://github.com/department-of-veterans-affairs/devops/pull/6427). After setting `mock_search` to false, one can see there is still connectivity / a response is returned from the search service.

File: Dockerfile [hunk 1]
```diff
@@ -86,6 +83,5 @@ ENV RAILS_ENV=production
 COPY --from=builder $BUNDLE_APP_CONFIG $BUNDLE_APP_CONFIG
 COPY --from=builder --chown=vets-api:vets-api /srv/vets-api/src ./
 COPY --from=builder --chown=vets-api:vets-api /srv/vets-api/clamav/database ../clamav/database
-RUN if [ -d certs-tmp ] ; then cd certs-tmp ; for i in * ; do cp $i /usr/local/share/ca-certificates/${i/pem/crt} ; done ; fi && update-ca-certificates
```
Comment (nathanhruby) [inline Dockerfile]: leave this for now, remove it after we have confirmed that the change in base stage is working and we have the devops repo builds set to use it

Comment (jbritt1) [reply Dockerfile]: Yep, wasn't going to merge this until everything was confirmed on the other side either way.

File: Dockerfile [hunk 2]
```diff
@@ -86,6 +83,6 @@ ENV RAILS_ENV=production
...
 USER vets-api
 ENTRYPOINT ["/usr/bin/dumb-init", "--", "./docker-entrypoint.sh"]
+RUN if [ -d certs-tmp ] ; then cd certs-tmp ; for i in * ; do cp $i /usr/local/share/ca-certificates/${i/pem/crt} ; done ; fi && update-ca-certificates
```
Comment (nathanhruby) [inline Dockerfile]: this is in the wrong place, needs to be before the USER

Comment (jbritt1) [reply Dockerfile]: I put it back in the same place it was before this PR. Are you saying it was already wrong? Or that I introduced something incorrect in trying to add it back?

Comment (jbritt1) [reply Dockerfile]: My bad, think I fixed it now.

**Comment by jbritt1** [comment]
Finally have this working in sandbox, dev, and staging at this time so I think we are good to roll this out after https://github.com/department-of-veterans-affairs/devops/pull/6479 is merged. We can make another pass to actually remove the duplicative action after a day or two of monitoring or something.

**Comment by jbritt1** [comment]
https://dsva.slack.com/archives/CBU0KDSB1/p1582724947130100

**Pull request merged**